player/lua/console.lua

```diff
@@ -1990,9 +1990,16 @@ mp.register_script_message('get-input', function (script_name, args)
 
     if args.items then
         selectable_items = {}
+
+        -- Limit the number of characters to prevent libass from freezing mpv.
+        -- Not important for terminal output.
+        local limit = terminal_output() and 5000
+                      or (5 * mp.get_property_native('osd-width') / opts.font_size)
```

Comment:
it could be `opts.font_size * 0.6` for width. I think 0.6 is ratio is common. But either way, does it has to be 5 times? Seems bit much?